config/config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Do you need it? Seems unused

Reply:
yeah, I need it to be able to connect to postress service, but I actually only need development environment for this task.

Reply:
I need it to be able to use my models 
you can check `api/models/index.js` file